### PR TITLE
fix doc warnings

### PR DESCRIPTION
**Pull request by nden**
Fixes several warnings when building the documentation with Sphinx.

### CODE REVIEW

File: docs/jwst/tweakreg/README.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Description
+x1Description
```
[the remark this answers is not in the document]
Comment (nden) [reply docs/jwst/tweakreg/README.rst]: It was probably an unsuccessful Ctr-x-1 in emacs. Good catch.

**Comment by hbushouse** [comment]
Related to #2675

**Comment by nden** [comment]
I am going to merge this and attempt fixing the duplicate warnings in another PR.

**Comment by nden** [comment]
BTW this fixed ~250 warnings.

File: .rtd-environment.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ dependencies:
   - numpy
   - scipy
   - pip:
+    - sphinxcontrib-programoutput
```
[the remark this answers is not in the document]
Comment (nden) [reply .rtd-environment.yml]: This runs and prints out the output of a command, like here

https://jwst-pipeline.readthedocs.io/en/latest/jwst/exp_to_source/main.html#usage

[the remark this answers is not in the document]
Comment (pllim) [reply .rtd-environment.yml]: The warning might confuse user though, otherwise looks cool!

[the remark this answers is not in the document]
Comment (nden) [reply .rtd-environment.yml]: Never mind the warning. I pasted the output of my environment. There is no warning. :smiley:

**Pull request merged**